### DISCUSSION TITLE
IR-Illumination set up

### DISCUSSION THREAD

**Discussion by atasevac** [openPupil/Open-PupilEXT#5] [General]
Hi, our team is interested in setting up the PupilEXT platform and we are very excited to get it up and running! We are looking over the components to order and we're wondering if there is a specific IR Illuminator you recommend for us to use?

**Comment by BZandi** [comment]
When it comes to IR illumination, the main question is which wavelength to choose. The answer mainly depends on two points:

1) First, you have to check the spectral sensitivity of the camera sensor in the IR range. It is recommended to use a monochrome camera, as the colour cameras usually have IR-blocker on their sensors, which reduces the sensitivity for IR light.

2) In pupillometry applications, a NIR colour filter is used in front of the camera's lens optic to block the visible spectrum range below ~780nm as far as possible, leading to less reflections in the eye's cornea and better pupil detection. Therefore, the selection of the IR light source also depends on the transmission behaviour of the selected filter.

For example, we used a Basler acA1300-200um camera in one of our demo setups and placed a Schneider IF 092 SH 27.0 (see Link 1) filter in front of the camera's lens. We used LEDs with a peak wavelength of 850 nm (SFH- 4715AS from Osram) as an IF light source on a custom-built PCB board (see Link 2, designed with Eagle). I assume that you want to buy a "ready to go" commercial IR-light source, then you should only pay attention to the peak wavelength.

Basically, the higher you go with the peak wavelength of the IR LEDs, the more power you need for the LEDs to produce enough brightness in the captured image. Of course, you could increase the camera's exposure time or open the aperture of the camera lens to compensate, but this could have negative side effects on the frame rate and the sharpness of the image.

Link 1: https://schneiderkreuznach.com/application/files/4215/7302/9820/Datasheet-color-filter-en.pdf
Link 2: https://github.com/openPupil/Open-PupilEXT/tree/main/IR_PCB
